janus_server/src/datastore.rs

```diff
@@ -615,6 +615,63 @@ impl<C: Clock> Transaction<'_, C> {
             .collect::<Result<Vec<Nonce>, Error>>()
     }
 
+    /// get_unaggregated_client_report_nonces_by_collect_for_task returns pairs of nonces and
+    /// aggregation parameters, corresponding to client reports that have not yet been aggregated,
+    /// or not aggregated with a certain aggregation parameter, and for which there are collect
+    /// jobs, for a given task.
+    #[cfg(test)]
+    #[tracing::instrument(skip(self), err)]
+    pub async fn get_unaggregated_client_report_nonces_by_collect_for_task<
```

Comment:
I think the doccomment here and on `get_unaggregated_client_report_nonces_for_task` should be clarified to explain that one should only be used for VDAFs with no aggregation parameter.